modules/installation-cloudformation-subnet-localzone.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+[id="installation-cloudformation-subnet-localzone_{context}"]
+= CloudFormation template for the subnet that uses AWS Local Zones
+
+You can use the following CloudFormation template to deploy the subnet that
+you need for your {product-title} cluster that uses AWS Local Zones.
+
+.CloudFormation template for the subnet
+[%collapsible]
+====
+[source,yaml]
+----
```

Comment:
Yes. @mtulio, do you have a separate task to track getting these added to the /installer repo? I can add a subtask to the docs follow-ups story, if you'd like.

Reply:
@katherinedube I opened a PR last year to minimal/chore improvements on the existing CloudFormation templates on installer, but it wasn't reviewed.

https://github.com/openshift/installer/pull/6088

The Phase-1 of Local Zones development should be required and has installer changes. I will add it to the same PR as it is part of the same solution:

https://github.com/openshift/installer/pull/6371/files#diff-f2506b1921b9d250fcbb846a9c79caca23fab3ef60f245085d1389639e149c75

As the PR of the installer for Phase-1 is taking a long to review, feel free to share the needed to create a separate PR for that. (as we are delivering this now, I think it could be a good idea)

Reply:
@mjpytlak, do you have a suggestion for who we should tag on the installer team about this?

Reply:
@kalexand-rh : @patrickdillon